kustomize/overlays/dev/secrets/event-bridge-manager-secrets.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   name: event-bridge-manager-secrets
 spec:
   encryptedData:
-    EVENT_BRIDGE_KAFKA_BOOTSTRAP_SERVERS: AgBdTPopaJ4pLj8tJH568uMi88pMVRo+A0wy5+JpWnGYUvledP0AaUFtl4eI9jZY1LqQRpebcLFTV5b/OoLFDaPMLYfB2ix3FBQOM6TsiBFhgOxjOrD4ucbjD1t2JMFEGT+lVXfVSLZsAgKFTK40nGIFa63nXWgzWxicVO6JCbWmVX25nlXnJjRNuPaMSFAtJBV7WqoEjHjb0fEWqoN5sUw0Lk1bmVoF2hOPQwYlUwZjKdss8tvvxYGX/0kJDHPna/3zhSPBFQ3HvJiwkZru/e9X3Ha87/BAQrOXDzS+TosZf+nHF9KTY3POCygtzfy/t88Pja1yzvdUjvLmvAQYfEJ57AtlGe3VAJPHyyholn1yCjyE4vzOu5EZC96OSQSPP805PPXZu++P4BtAMTdo6EkDVe29vxLS7AtMNaYWusgOHSFSkryWTTKkJKPiHa6QUHiMfg7G/0AYpUij8mUZWTgONaYGEyRM0CfD7vTG2oeGE+qXboxMKrhOD4xYPn960/BW3xIxwGAJGgbk64viMTB9q9eixhqpm4WOYWvlpZmWZnD0Tnf2uaxhJ6YdTNhVNyC2I2+kki0KcMhFBKHROQXKQnU4jAUn4dNsrznGRuq93Qx0kUo6Z4BGGfVnqdrAbJgZdpb2sSkReMlG5F6fsGdkSWQs06DO9064pg+h6RYikNLJYtjVAjQ6UkZMTCUbzm6XoYVq8sAO9O7VW4jXQZR9uZIOV1EaEhMSVsQvoOFFoOlbldYWmqx3HQfQe/tEfL+Z0wjhAq8PJuJPD+7TXcsASA==
+    EVENT_BRIDGE_KAFKA_BOOTSTRAP_SERVERS: AgA4f35assjSnt8aUT5ONa3TZfzIxoTGMN7ja41nB9SGap7j+eWus2LUWIMQaNdvaD7p7tYW8EpTPEBCDAN2MaMz4Dahky1TrhtmpKPCMZUOjiAdi5PsT5h31a14wM6H2Ad8J5EZFZ7CPLC/9WTo6N4KhrUrrT0rzV7ufLwvax1U/7YyySv+lsWmSYASFcSP+9vu1FC/wzoKiUzWlGpCMZmF50zlU5XJrmKy5jmmFTI9CwCRKyzQN0fTB2c2FcKVYd/meMBSOxXi9lURDcSt8XnPLLbDxVayfNGq+o9Giqm3uEo6NmtccgLiU14Xo8tmRa/W+3WkOttbQ91/CpIlrH7hqMgEUPSw8vHLiT7WDPOKTuZAWqiy4EHlW5rrpSp3cY08IbnBnivwBmeux3iTipyAbrnSTuv73/e9ihOVQnIsD3hYJIkJkVj8/afe+hidzH19W/SxDw/9Nba8tI8DVTYHSCdpHIFmzvq/gSqk6FdL/F2s3iw7o6S4eu5f2n5fSbuLmRnMj0NVskkruwuufzVttIIv2zagpMZ5I8SZ4KzvRiE3cXJLmIPW+VRo8aqjds62HFzUpwef/N9FgWxcFwOYT6+6HqZ4OuBNUfrrzzFuiPpezzPDLaC9zkQzkzdbzFvNaSFXK37LYPj4xRLUuBv57Uf49xnB/eHbQ4JZbT1klv2F9+a0z1TqjYpiy80jkWzb0sZCTq91tirOWIjcBoH/c9fUNkqyrXrosSmtmmj4xwdrKsveUI+jSwPFXFcSvj8oC1Omf1o579M=
```

Comment:
because at the moment `EVENT_BRIDGE_KAFKA_BOOTSTRAP_SERVERS` starts with `https` but knative does not accept it, as kafka does not use an `https` protocol